recipes/capnproto/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+CMAKE_CXX_FLAGS="-fPIC"
+if [ "$(uname)" = "Darwin" ]; then
+    # "-stdlib=libc++ -mmacosx-version-min=10.7" are required to enable C++11 features
+    CMAKE_CXX_FLAGS="$CMAKE_CXX_FLAGS -stdlib=libc++ -mmacosx-version-min=10.7"
+fi
+
+cmake \
+    -DBUILD_TESTING=OFF \
```

Comment:
We should try testing in these builds though. I understand Windows is broken, but this should still work. Right?